components/molecule/quickAction/src/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const CLASS = 'sui-MoleculeQuickAction'
+
+const SIZES = {
+  LARGE: 'large'
```

Comment:
IMHO the default behavior should use one of the sizes. The `S` or the `M`.

The reason? We should see the `size` as states of the component. It should be controlled. In case it's not, it increases the difficulty of the maintainability and the accidental complexity.